include/picongpu/fields/incidentField/profiles/GaussianPulse.hpp

```diff
@@ -316,43 +316,43 @@ namespace picongpu
                 } // namespace detail
 
                 template<typename T_Params>
-                struct GaussianBeam
+                struct GaussianPulse
                 {
                     //! Get text name of the incident field profile
                     static HINLINE std::string getName()
                     {
                         // This template is used for both Gaussian and PulseFrontTilt, distinguish based on tilt value
                         using TiltParam = detail::TiltParam<T_Params>;
                         bool isTilted = (std::abs(TiltParam::TILT_AXIS_1) + std::abs(TiltParam::TILT_AXIS_2) > 0);
-                        return isTilted ? "PulseFrontTilt" : "GaussianBeam";
+                        return isTilted ? "PulseFrontTilt" : "GaussianPulse";
                     }
                 };
 
             } // namespace profiles
 
             namespace detail
             {
-                /** Get type of incident field E functor for the gaussian beam profile type
+                /** Get type of incident field E functor for the GaussianPulse profile type
                  *
                  * @tparam T_Params parameters
                  */
                 template<typename T_Params>
-                struct GetFunctorIncidentE<profiles::GaussianBeam<T_Params>>
+                struct GetFunctorIncidentE<profiles::GaussianPulse<T_Params>>
                 {
-                    using type = profiles::detail::GaussianBeamFunctorIncidentE<T_Params>;
+                    using type = profiles::detail::GaussianPulseFunctorIncidentE<T_Params>;
                 };
 
-                /** Get type of incident field B functor for the gaussiam beam profile type
+                /** Get type of incident field B functor for the  GaussianPulse profile type

```

Comment:
There is an extra space here between `the  GaussianPulse`

Reply:
Removed.